src/character.cpp

```diff
@@ -4878,14 +4878,16 @@ void Character::temp_equalizer( body_part bp1, body_part bp2 )
     temp_cur[bp1] += diff;
 }
 
-Character::comfort_level Character::base_comfort_value( const tripoint &p, bool message ) const
+Character::comfort_response_t Character::base_comfort_value( const tripoint &p ) const
 {
     // Comfort of sleeping spots is "objective", while sleep_spot( p ) is "subjective"
     // As in the latter also checks for fatigue and other variables while this function
     // only looks at the base comfyness of something. It's still subjective, in a sense,
     // as arachnids who sleep in webs will find most places comfortable for instance.
     int comfort = 0;
 
+    comfort_response_t comfort_response;
+    comfort_response.aid = &item( "null" );
```

Comment:
This stores a pointer to a temporary object. This pointer becomes invalid right after this statement. I suggest you initialise it with a `nullptr` and check for that.

Reply:
Yep, it should be fixed now.